data/kamailio.cfg

```diff
@@ -1306,87 +2086,100 @@ route[CHECK_STATS] {
 		jansson_set("string", "timestamp", "$var(isotime)", "$var(tmp_json)");
 		jansson_set("string", "name", "$var(key)", "$var(tmp_json)");
 		jansson_set("integer", "value", "$shtitval(i1)", "$var(tmp_json)");
-		
+
+		#!ifdef TO_ELASTIC
 		$var(stats_generic) = $var(stats_generic) + " {\"index\": {\"_index\":\"homer-"+ $timef(%d.%m.%Y) +"\",\"_type\":\""+$var(type)+"\" }}\n";
 		$var(stats_generic) = $var(stats_generic) + $var(tmp_json) + "\n";
+		$var(tmp_json) = 0;
+		#!endif
 
+		#!ifdef TO_GRAYLOG
+		jansson_set("string", "timestamp", ""+$var(gentime)+"."+$TV(u)/999+"", "$var(tmp_json)");
+		jansson_set("string", "short_message", "Homer-Stats", "$var(tmp_json)");
+		$var(graylog_json) = $var(tmp_json);
 		$var(tmp_json) = 0;
-		$sht(a=>method::total) = 0;
+		route(SEND_ES_DATA);
+		$var(graylog_json) = 0;
+		#!endif
+
 	}
 	sht_iterator_end("i1");
 	sht_rm_name_re("d=>.*");
-	
-	$var(body) = $var(stats_generic);
 
+	$var(body) = $var(stats_generic);
 	route(SEND_ES_DATA);
 #!endif
 }
 
 route[SEND_ES_DATA] {
```

Comment:
maybe we could rename this route to something like SEND_HTTP_DATA (and the reply accordingly).